1-exercises/A-this/exercise-3.js

```diff
@@ -8,8 +8,12 @@
 
 let bankAccount = {
   currentAmount: 100,
-  deposit: function (amount) {},
-  withdraw: function (amount) {},
+  deposit: function (amount) {
+    this.currentAmount.deposit;
```

Comment:
Hi Miguel
you have done amazing job. what suggest is  deposit should be added to the currentAmount, and withdraw should be substract from the currentAmount.